Makefile

```diff
@@ -1083,40 +1075,48 @@ ui-lint: pkg/ui/yarn.installed $(UI_PROTOS) $(UI_PROTOS_CCL)
 
 # DLLs are Webpack bundles, not Windows shared libraries. See "DLLs for speedy
 # builds" in the UI README for details.
-UI_DLLS := pkg/ui/dist/protos.dll.js pkg/ui/dist/vendor.dll.js
-UI_MANIFESTS := pkg/ui/protos-manifest.json pkg/ui/vendor-manifest.json
-
-# (Ab)use a pattern rule to teach Make that this one command produces two files.
-# Normally, it would run the recipe twice if dist/FOO.js and FOO-manifest.js
-# were both out-of-date. [0]
+UI_CCL_DLLS := pkg/ui/dist/protos.ccl.dll.js pkg/ui/dist/vendor.oss.dll.js
+UI_CCL_MANIFESTS := pkg/ui/protos.ccl.manifest.json pkg/ui/vendor.oss.manifest.json
+UI_OSS_DLLS := $(subst .ccl,.oss,$(UI_CCL_DLLS))
+UI_OSS_MANIFESTS := $(subst .ccl,.oss,$(UI_CCL_MANIFESTS))
+
+# (Ab)use pattern rules to teach Make that this one Webpack command produces two
+# files. Normally, Make would run the recipe twice if dist/FOO.js and
+# FOO-manifest.js were both out-of-date. [0]
 #
-# XXX: Ideally we'd scope the dependency on $(UI_PROTOS) to the protos DLL, but
-# Make v3.81 has a bug that causes the dependency to be ignored [1]. We're stuck
-# with this workaround until Apple decides to update the version of Make they
-# ship with macOS or we require a newer version of Make. Such a requirement
+# XXX: Ideally we'd scope the dependency on $(UI_PROTOS*) to the protos DLLs,
+# but Make v3.81 has a bug that causes the dependency to be ignored [1]. We're
+# stuck with this workaround until Apple decides to update the version of Make
+# they ship with macOS or we require a newer version of Make. Such a requirement
 # would need to be strictly enforced, as the way this fails is extremely subtle
 # and doesn't present until the web UI is loaded in the browser.
 #
-# [0]: https://stackoverflow.com/a/3077254/1122351
-# [1]: http://savannah.gnu.org/bugs/?19108
-.SECONDARY: $(UI_DLLS) $(UI_MANIFESTS)
-pkg/ui/dist/%.dll.js pkg/ui/%-manifest.json: pkg/ui/webpack.%.js pkg/ui/yarn.installed $(UI_PROTOS) $(UI_PROTOS_CCL)
-	$(NODE_RUN) -C pkg/ui $(WEBPACK) -p --config webpack.$*.js
+# [0]: https://stackoverflow.com/a/3077254/1122351 [1]:
+# http://savannah.gnu.org/bugs/?19108
+.SECONDARY: $(UI_CCL_DLLS) $(UI_CCL_MANIFESTS) $(UI_OSS_DLLS) $(UI_OSS_MANIFESTS)
+
+pkg/ui/dist/%.oss.dll.js pkg/ui/%.oss.manifest.json: pkg/ui/webpack.%.js pkg/ui/yarn.opt.installed $(UI_PROTOS_OSS)
```

Comment:
i don't think these targets should need `yarn.opt.installed`, just `yarn.installed`?